.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Description
+This PR…
```

Comment:
Use [Github keywords](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) to link to a related issue, e.g. `fixes #123` or `closes #123`